scripts/code-connect/generate-connect-file-contents.ts

```diff
@@ -0,0 +1,90 @@
+import ts from "typescript";
+import { ZDS_ASSETS_FILE_ID } from "../../figmaConfig.js";
+
+export const generateConnectFileContents = (iconName: string, id: string) => {
+  return ts.factory.createSourceFile(
```

Comment:
i dont understand this and i dont like it

Reply:
just promise it works and i'll approve the pr

Reply:
i promise but you'll be even more confused when you see the web components code connect script